content/applications/hr/payroll.rst

```diff
@@ -457,22 +709,101 @@ advantage listed for a particular structure type is only available for that spec
+
+Display section
+~~~~~~~~~~~~~~~
+
+- :guilabel:`Show Name`: tick the checkbox to have the benefit name appear in the salary package
+  configurator.
+- :guilabel:`Display Type`: select from the drop-down menu how this benefit is displayed. The
+  options are :guilabel:`Always Selected`, :guilabel:`Dropdown`, :guilabel:`Dropdown Group`,
+  :guilabel:`Slider`, :guilabel:`Radio Buttons`, :guilabel:`Manual Input`, or :guilabel:`Text`. This
+  field is required.
+
+  Depending on the selection made, additional configurations need to be made. For example, if
+  :guilabel:`Radio Buttons` is selected, the individual radio buttons must be entered.
+- :guilabel:`Icon`: an icon from the Font Awesome library can be visible for this benefit. Enter the
+  text code for the icon in this field. For example, to display a suitcase icon, the code `fa
+  fa-suitcase` is entered on this line.
+- :guilabel:`Hide Description`: tick the checkbox to hide the benefit description if the benefit is
+  not selected by the employee.
+- :guilabel:`Folded`: if the benefit should be folded, or hidden, because it is dependant on another
+  benefit selection, tick the checkbox. The following fields appear when this is active:
+
+  - :guilabel:`Fold Label`: enter a name for the folded section of the benefit.
+  - :guilabel:`Fold Res Field`: select the contract field this benefit is tied to using the
+    drop-down menu. If this field is selected on the contract, then this benefit becomes visible.
+
+Activity section
+~~~~~~~~~~~~~~~~
+
+- :guilabel:`Activity Type`: select the activity type that is automatically created when this
+  benefit is selected by the employee, from the drop-down menu.
+- :guilabel:`Activity Creation`: select when the activity is created, either when the
+  :guilabel:`Employee signs his contract`, or when the :guilabel:`Contract is countersigned`. Click
+  the radio button next to the desired selection.
+- :guilabel:`Activity Creation Type`: select the parameters for when the activity is created, either
+  :guilabel:`When the benefit is set`, or :guilabel:`When the benefit is modified`. Click the radio
+  button next to the desired selection.
+- :guilabel:`Assigned to`: select the user the activity is automatically assigned to, using the
+  drop-down menu.
+
+Sign section
+~~~~~~~~~~~~
+
+- :guilabel:`Template to Sign`: if the employee is required to sign a document when selecting this
+  benefit, select the document template from the drop-down menu.
+
+  For example, a benefit regarding the use of a company car may require the employee to sign a
+  document acknowledging the company's car policies.
+
+Description tab
+~~~~~~~~~~~~~~~
+
+Provide any additional information in this tab to help clarify the benefit.
+
+.. image:: payroll/new-benefit.png
    :align: center
-   :alt: List of advantages employee's can have.
+   :alt: A new benefit form filled out for an internet subscription.
```

Comment:
is this screenshot in the correct spot? Is this a shot of the Description tab? If not, I'd suggest moving it to a more appropriate place in the doc. Thanks! 👍

Reply:
I typically put the form at the bottom of all the explanations for the form, but I moved it to under the general/top section.